aquasec/provider.go

```diff
@@ -63,6 +64,12 @@ func Provider(v string) *schema.Provider {
 				DefaultFunc: schema.EnvDefaultFunc("AQUA_CONFIG", "~/.aquasec/tf.config"),
 				Description: "This is the file path for Aqua provider configuration. The default configuration path is `~/.aqua/tf.config`. Can alternatively be sourced from the `AQUA_CONFIG` environment variable.",
 			},
+			"cloud_env": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("AQUA_CLOUD_ENV", nil),
+				Description: "If present, this variables signifies which environment of Aqua Cloud we are using. The default configuration path is `~/.aqua/tf.config`. Can alternatively be sourced from the `AQUA_CLOUD_ENV` environment variable.",
```

Comment:
The sentence in this description referencing default configuration path is not relevant for this variable. It's just left over from copy pasting the config variable description above.

Reply:
Thanks for the catch and review comment. Fixed it.